content/Google is a modal dialog - Martin Rubi.md

```diff
@@ -0,0 +1,116 @@
+## Google is a modal dialog
+
+### The shame of being a modal dialog
+
+A modal dialog is a window which pops up on top of every other window freezing the
+underlaying user interface until the user dismisses it.
+
+Its use was quite frequent in the early days of desktop windows apps but with the
+beginning of browser user intefaces they not only became obsolete but some articles
+were writen discouraging its use in favor of edit in place interfaces.
+
+I'm no UX expert but I am a user of graphics user intefaces on a daily basis and
+ as a user I will speak up in defense of modal dialogs when used properly.
+
+### The importance of being modal
+
+What do modal dialogs do?
+
+When I first thought in modal dialogs I focused mainly on the space location
+in the user interface.
+
+The first desktop apps achieved 3D emulation by overlapping 2D rectangles called
+windows [1].
+
+Browser interfaces avoid 3D emulation and usually stick to a strict 2D interface
+through laying out its elements instead of using overlapping.
+
+However this is just the implementation of what modal dialogs do and not its functional
+purpose and value.
+
+Modal dialogs interrupt the flow of the user attention to solve a particular task
+providing a self contained, highly focused functional and visual context for the user.
+
+That is they provide for a very short period of time a user interface for the user
+to solve a specific task hidding every other complexity not related to that task.
+
+Once the user ends solving that specific part of the app it returns to the previous
+flow just like she left it.
+
+Whichs is pretty much the same workflow that we use to google things
+
+### Google is a modal dialog
+
+How do we google things?
+
+Usually we are doing something like writing a blog post and we find the need to
+search for information.
+
+We open a new tab with a clean and very specific design to do just one thing,
+search things.
+
+We select a few results, decide whether our question was answered or not and get
+back to our previous task closing the search tab.
+
+That is what modal dialogs functional purpose is regardless of its implementation
+on the user interface space.
+
+### Other issues with edit in place
+
+There are two very important functions in a modal dialog.
+
+The first one would be the one already mentioned, to provide an isolated context
+to solve a specific task hidding all other context not relevant to that task.
+
+Yet there is another function just as important: hidding unnecessary complexity
+in the main application interface until that complexity is required.
+
+The rationale of a modal dialog is to present to the user the complexity of
+a user interface when the user needs it, not before, not afterwards, and just for
+the time the user is dealing with that task.
+
+The proliferation of menu options and visual aids can hurt more than they would help
+producing a sense of overwhelming and confusion in the user.
```

Comment:
"more than it can help"*. With "can hurt" you are referring to "The proliferation" so the next part needs to refer to the same thing and use the correct pronoun.
If your intention was to focus on "menu options and visual aids", I suggest you rephrase the sentence like follows:
"Menu options and visual aids can hurt more than they can help"

Reply:
I would also add a comma after "help" helps separate the sentence into more readable chuncks.

Reply:
"a sense of overwhelm* and confusion". Overwhelming is an adjective but was not being applied to anything.